using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md

```diff
@@ -80,21 +82,31 @@ public void onNewToken(String token) {
 {% endtab %}
 
 {% tab title="iOS" %}
-To get your app ready to handle push notifications, follow [Apple's instructions](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/HandlingRemoteNotifications.html#//apple_ref/doc/uid/TP40008194-CH6-SW1). After you receive the APNS token, send this token and the device information to Campaign Classic using the `registerDevice` API.
-
-### registerDevice
+{% hint style="info"}
+To get your app ready to handle push notifications, see the tutorial on [configuring remote notification support](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/HandlingRemoteNotifications.html#//apple_ref/doc/uid/TP40008194-CH6-SW1). After you receive the Apple Push Notification service (APNs) token, send this token and the device information to Campaign Classic using the `registerDevice` API.
 
 The `registerDevice` API registers a device with your Campaign Classic registration server. It takes the APNS token as a parameter with a user key that identifies a user, such as an email address or a login name. You can also provide a map of the custom key-value pairs that you want to associate with the registration. A boolean value is returned in the callback, which signals whether the registration was successful.
+{% endhint %}
+
+### Swift
```

Comment:
Have we done this change in other extension docs too? I find this format a little odd
```
Swift -> Example
Objective-C - > Syntax, Example
```
(1) We should include syntax with Swift too; or 
(2) Keep syntax first (Objective-C only in case of 2.x) and then include Swift/ Objective-C examples. Refer to - https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#displayedlocations

Thoughts?

Reply:
Yeah, I should also include the syntax for it - that being said, I didn't have the syntax, which is why I wasn't sure what to add.